test/test_rule_set_expanding_shorthand.rb

```diff
@@ -242,37 +241,37 @@ def test_getting_background_gradient_from_shorthand
   # List-style shorthand
   def test_getting_list_style_properties_from_shorthand
     expected = {'list-style-image' => 'url(\'chess.png\')', 'list-style-type' => 'katakana',
-      'list-style-position' => 'inside'}
+                'list-style-position' => 'inside'}
 
     shorthand = "list-style: katakana inside url(\'chess.png\');"
     declarations = expand_declarations(shorthand)
     assert_equal expected, declarations
   end
 
   def test_getting_list_style_position_from_shorthand
-    ['inside', 'outside'].each do |position|
+    %w[inside outside].each do |position|
```

Comment:
prefer the old style since it's simple-stupid ... but not sure if the rest of the code used it ... maybe just disable if it's mixed :/
```suggestion
    ['inside', 'outside'].each do |position|
```

Reply:
It's mixed, I just used variant with less offences.